alcs-frontend/src/app/features/admin/configuration/configuration.component.scss

```diff
@@ -1,18 +1,73 @@
 @use '../../../../styles/colors';
 
-.value {
+.row {
+  display: flex;
+  margin: 24px 0;
+  width: 100%;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.label-input-container {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  width: 100%;
+}
+
+.toggle-label {
+  font-weight: 700;
+  flex-grow: 1;
+}
+
+.toggle {
   display: flex;
   align-items: center;
+  gap: 4px;
+  color: colors.$black;
+  font-weight: 700;
 }
 
-.subheading2 {
-  margin-top: 36px !important;
+.inactive {
+  color: colors.$grey;
+  font-weight: 400;
 }
 
-.editing {
-  max-width: 600px;
+:host::ng-deep {
+  .toggle {
+    mat-label {
+      margin-right: 8px;
+    }
+
+    & .mdc-form-field {
```

Comment:
comment: I'm surprised we are doing so much CSS for Admin when historically its been a dont care

Reply:
I think we didn't do much CSS when there were no designs for admin features. This one actually has a Figma design so I styled it according to that

Reply:
Removed some more unused styling